data_pipeline/gmail_data/get_gmail_data.py

```diff
@@ -0,0 +1,45 @@
+from typing import Iterable
+import pandas as pd
+
+from google.oauth2 import service_account
+from googleapiclient.discovery import build
+from googleapiclient.discovery import Resource
+
+
+# this connection type will be changed !!
+def connect_to_email(userId: str) -> Resource:
+    SCOPES = ['https://www.googleapis.com/auth/gmail.readonly']
+    SERVICE_ACCOUNT_SECRET_FILE = 'DELETE/elife-data-pipeline-test.json'
```

Comment:
perhaps make this configurable, maybe in this case it makes sense if the path to the file is passed in from the DAG?